common/lib/xmodule/xmodule/js/src/video/05_video_quality_control.js

```diff
@@ -5,23 +5,19 @@
 define(
 'video/05_video_quality_control.js',
 [],
-function () {
+function (HtmlUtils) {
     var template = [
         '<button class="control quality-control is-hidden" aria-disabled="false" title="',
             gettext('High Definition'),
         '">',
-            '<span class="icon-fallback-img">',
-                '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
+            '<span class="icon icon-hd" aria-hidden="true">HD</span>', // "HD" is treated as a proper noun
```

Comment:
I am surprised that this concatenation of HTML (which includes translated parts) is not being caught by the linter. It should be built up using HTMLUtils so that the translated parts can be properly escaped.

FYI @robrap